discord/app_commands/commands.py

```diff
@@ -2094,6 +2094,22 @@ def describe(**parameters: Union[str, locale_str]) -> Callable[[T], T]:
         async def ban(interaction: discord.Interaction, member: discord.Member):
             await interaction.response.send_message(f'Banned {member}')
 
+    Alternatively, you can describe parameters using Google, Sphinx, or Numpy style docstrings.
+
+    Example:
+
+    .. code-block:: python3
+
+        @app_commands.command()
+        async def ban(interaction: discord.Interaction, member: discord.Member):
+            '''
+            Parameters
+            -----------
+            member: discord.Member
+                the member to ban
+            '''
```

Comment:
```suggestion
            """
```

Reply:
If i use `"""` it will interfere with the outer docstring because that uses `r"""` to open

Reply:
You can escape the inner docstring's quotes by adding a backslash before each `"`

Reply:
Done